styleguide/_themes/global/scss/bootstrap_overrides.scss

```diff
@@ -31,6 +31,14 @@
   background-color: $brand-success;
 }
 
+//Forms
+.form-control.form-checkboxes,
+.form-control.form-radios {
```

Comment:
If they can be isolated and still work, this should git rid of sass-lint errors, and this code can be added to the forms.scss file instead of here.